tests/Dfc.CourseDirectory.Testing/DataStore/CosmosDb/QueryHandlers/GetApprenticeshipsByIdsHandler.cs

```diff
@@ -12,7 +13,7 @@ public class GetApprenticeshipsByIdsHandler
         public IDictionary<Guid, Apprenticeship> Execute(InMemoryDocumentStore inMemoryDocumentStore, GetApprenticeshipsByIds request) =>
             inMemoryDocumentStore.Apprenticeships.All
                 .Where(p => request.ApprenticeshipIds.Contains(p.Id))
-                .Where(p => p.RecordStatus != 4 && p.RecordStatus != 8)
+                .Where(p => p.RecordStatus != (int)RecordStatus.Archived && p.RecordStatus != (int)RecordStatus.Deleted)
```

Comment:
Better to use `ApprenticeshipStatus` here; `RecordStatus` is from the old models and shouldn't be used going forward

Reply:
Ah, maybe we should use Obsolete to signpost to the new one, or at least a comment on the old one. Thanks

Reply:
`[Obsolete]` will fail build currently

Reply:
Yeah, changed my mind immediately, will also be very noise if we obsolete everything in legacy.